src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -2062,6 +2062,58 @@ def math(self):
 
         return MapdlMath(self)
 
+    @property
+    def krylov(self):
+        """APDL krylov interface
+
+        Returns
+        -------
+        :class:`MapdlMath <ansys.mapdl.core.krylov.Krylov_functions>`
+
+        Examples
+        --------
+        Solving using krylov method
```

Comment:
I love this! But because ``mapdl.krylov`` returns a class, this docstring is not shown, rather the doc string of the class.

You can keep it if you wish, but there is such good content here, that you might want to copy/paste some over to the krylov class/docs/example ... as you wish.